backend/cmd/cluster_counter/main.go

```diff
@@ -5,45 +5,16 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	countModel "github.com/Avi18971911/Augur/pkg/count/model"
```

Comment:
Moved the VAST majority of the logic here to data_processor, which ought to do the cluster processing as well. basically it ought to be triggered via cron job